docs/Get started/FAQ.md

```diff
@@ -575,11 +575,20 @@ Fleet is tested with Redis 5.0.14 and 6.2.7. Any version Redis after version 5 w
 Most likely, yes! While we'd definitely recommend keeping Fleet up to date in order to take advantage of new features and bug patches, most legacy versions should work with Redis 6. Just keep in mind that we likely haven't tested your particular combination so that you may run into some unforeseen hiccups.
 
 ## What happened to the "Schedule" page? 
-Scheduled queries are not gone! Instead the concept of a scheduled query has been merged with a saved query. 4.35 implemented an automatic migration which transitions any pre-existing scheduled query and pack into the new merged query concept.
+Scheduled queries are not gone! Instead the concept of a scheduled query has been merged with a saved query. After 4.35, there will only be one page of queries and a query can be scheduled (via familiar attributes such as "interval," "platform") or it can simply be saved to be run ad-hoc. Queries can now belong to teams or they can be a global query which every team inherits. This greatly simplifies the mental model of the product and enables us to build [exciting features](https://github.com/fleetdm/fleet/issues/7766) on top of the new unified query concept. 
 
-Before 4.35, a scheduled query was a separate item treated at the same level as a query. After 4.35, there will only be one page of queries and a query can be scheduled or it can simply be saved to be run ad-hoc. Queries can now belong to teams or they can be a global query which every team inherits.
+To achieve the above, 4.35 implemented an automatic migration which transitions any pre-existing scheduled query and pack into the new merged query concept: 
```

Comment:
2017 packs were left untouched (`/packs/manage`), they still work the old way, where "schedules" added to the pack are references to queries.